Automattic/wp-calypso
# Woo Express: The state didn't update after upgrading an expired woo trial site

## Proposed Changes

* The state was not updated after upgrading an expired site.
* This would cause the user to be redirected back to the Trial Expired page, instead of the Trial Upgraded page.

## Testing Instructions

* Navigate to `/plans/my-plan/trial-expired/{YOUR-SITE}` on an expired woo trial site
* Upgrade your site by clicking on the "Upgrade now" button.
* Go through the checkout flow.
* You should no longer be redirected to the `plans/my-plan/trial-expired/{YOUR-SITE}`, you should see the Trial Upgraded page.

Closes #74785
